### PR TITLE
Stem regular Dutch participles

### LINKED ISSUE #384: Detect and stem Dutch participles
## Explanation
We should detect and stem regular Dutch participles

### PR DESCRIPTION
**Pull request by agnieszkaszuba**
## Summary

This PR can be summarized in the following changelog entry:

* Stems regular Dutch past participles

## Test instructions

This PR can be tested by following these steps:

* Add more examples to the specs. The cases that need to be tested are: 

1) regular participles without a prefix (e.g. _gedanst_),
2) regular participles with a separable prefix, i.e. one that is followed by _ge_ (e.g. _**af**gezet_)
3) regular participles with an inseparable prefix, i.e. one that does not require _ge_ in front of it (e.g. _**be**antwoord_)
4) words that are non-participle exceptions (are on the nonParticiples exception list **or** begin with ge- and end with -heid/-tijd/-teit),
5) Participles with a prefix that is normally separable, but in this word acts as inseparable. For example, the prefix _aan_ is usually separable, but in the participle _aanschouwd_ it is inseparable  (see the inseparableCompoundVerbs array in morphology data file for the full list of those participles). 
6) Participles that are the same as the stem of the word, e.g. _omlijst_. See inseparableCompoundVerbsNotToBeStemmed in morphology data for full list.

This issue aims to cover mainly regular participles, so irregular participles falling outside of the checks described in points 5 and 6 may not (and don't have to be, for now) stemmed correctly.

## Quality assurance

* [x] I have tested this code to the best of my abilities
* [x] I have added unittests to verify the code works as intended

Fixes #384

### CODE REVIEW

File: packages/yoastseo/src/morphology/dutch/detectAndStemRegularParticiple.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	return null;
+};
+
+/**
+ * Checks whether the word is on an exception list of participles with an inseparable prefix that normally
+ * is separable, and stems it if found on the list.
+ *
+ * @param {array} dataInseparableExceptions The list of the exception words.
+ * @param {string} word 	The (unstemmed) word to check.
+ * @returns {null|string} The stemmed word or null if the word was not found on the exception list.
```
Comment (hansjovis) [inline packages/yoastseo/src/morphology/dutch/detectAndStemRegularParticiple.js]: Tiny formatting issue: an extra line above the return statement 🙂. 
```suggestion
 *
 * @returns {null|string} The stemmed word or null if the word was not found on the exception list.
```

File: packages/yoastseo/src/morphology/dutch/detectAndStemRegularParticiple.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	return null;
+};
+
+/**
+ * Detects whether a word is a regular participle and if so, returns the stem.
+ *
+ * @param {Object}  morphologyDataVerbs The Dutch morphology data for verbs.
+ * @param {string}  word                The word (not stemmed) to check.
+ *
+ * @returns {string} The participle stem or null if no regular participle was matched.
```
Comment (hansjovis) [inline packages/yoastseo/src/morphology/dutch/detectAndStemRegularParticiple.js]:
```suggestion
 * @returns {string|null} The participle stem or null if no regular participle was matched.
```

**Comment by nataliashitova** [comment]
@agnieszkaszuba could you take another look at compatibility between `javascript` and `premium-configuration` branches and return this PR to the Lingo process after that?

**Comment by agnieszkaszuba** [comment]
> @agnieszkaszuba could you take another look at compatibility between `javascript` and `premium-configuration` branches and return this PR to the Lingo process after that?
> 
> Right now I see that some exceptions for the last rule (usually separable suffixes being unseparable) are being moved to a separate class in the morphologyData file, which class is never used in your `javascript` branch.

This is fixed now :)

**Comment by manuelaugustin** [comment]
Acceptance 👍

**Pull request merged**